src/components/Auth/Auth.js

```diff
@@ -0,0 +1,67 @@
+/* eslint-disable react/prefer-stateless-function */
+import React, { Component } from 'react';
+// eslint-disable-next-line import/no-named-as-default
+import Signup from './Signup';
+import Login from './Login';
+import logo from '../../assets/images/world.svg';
+import backImage from '../../assets/images/jon-tyson-762619-unsplash.jpg';
+
+import '../../assets/css/signup.css';
+
+class SignUp extends Component {
+  state = {
+    auth: 'login'
```

Comment:
Since this `auth` key is changing the form to be displayed, give it a better name. It can be something like `formType` or `displayForm`.

Reply:
You changed the class instead of the auth key. I meant the auth o line 13.

Reply:
Thank you for the feedback. However, I would like you to shade more light on whether I should also change the route from being `/login` or `/register` to only be `/auth` for both those logging in or registering.